assets/css/v2/lcd_common/departures/later_departures.scss

```diff
@@ -0,0 +1,83 @@
+@use "sass:math";
+
+.later-departures {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  min-height: 280px;
+  background-color: $warm-neutral-90;
+
+  &__header {
+    box-sizing: inherit;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 0 32px;
+    border-bottom: 4px solid $true-grey-70;
+
+    h3 {
+      box-sizing: inherit;
+      flex-shrink: 0;
+      padding: 18px 0;
+      margin: 0;
+      font-family: Inter;
+      font-size: 40px;
+      font-weight: 500;
+      line-height: 43px;
+      color: $cool-black-15;
+      text-transform: uppercase;
+      letter-spacing: 2px;
+    }
+
+    ol {
+      display: flex;
+      flex-direction: row;
+      gap: 24px;
+      justify-content: space-between;
+      list-style: none;
+
+      .route-pill {
+        position: static;
+        box-sizing: border-box;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: auto;
+        min-width: 86px;
+        height: 44px;
+      }
+
+      .route-pill--yellow {
+        box-shadow: 1px 1px 1px 0 rgba(0 0 0 / 25%);
+      }
+
+      .route-pill__text {
+        position: static;
+        padding: 0 14px;
+        font-size: 36px;
+        line-height: 36px;
+      }
+    }
+  }
+
+  &--departure--selected {
+    position: relative;
+
+    &::after {
+      $carot-size: 16px;
```

Comment:
I think you meant `caret`? 🥕

Reply:
I did! I _vividly_ remember fixing this misspelling already but alas my memory is fallible 🙃